### ISSUE TITLE
Not working on Windows 10

### ISSUE THREAD

**Issue by larrybahr** [mrcoles/bookmarklet#7]
When using this with any code (including running test/run.sh) I am getting the following in the output file:
```javascript
javascript:(function()%7BNaN%7D)()
```

**Comment by richardfrost** [comment]
I am getting the same result on macos:

```js
javascript:(function()%7BNaN%7D)()
```

When I run it on the js file I want to turn into a bookmarklet, I get:

```js
javascript:(function()%7Bundefined%7D)()
```

**Comment by markhicken** [comment]
I'm also getting the same error.

**Comment by mrcoles** [comment]
Should be solved form #8 🥂
